docs/source/developers/guide/tutorials/r_tutorial.rst

```diff
@@ -24,4 +24,26 @@
 
 **********
 R tutorial
-**********
\ No newline at end of file
+**********
+
+The first R package tutorial to be included in the New Contributor's
+guide is a **Walkthrough** added in the **Writing Bindings**
+vignette. With time we will try to include additional tutorials
+directly into this guide.
+
+Writing Bindings Walkthrough
+----------------------------
+
+This tutorial will show how to do a binding of a C++ function
+`starts_with() <https://arrow.apache.org/docs/cpp/compute.html#containment-tests>`_
+to the (base) R function ``startsWith()``.
+
+To view the tutorial follow
+`this link <https://arrow.apache.org/docs/r/articles/developers/bindings.html#walkthrough>`_
+or run the following command in the R console:
+
+.. code-block:: R
+
+   vignette("bindings", package = "arrow")
```

Comment:
Unfortunately, this will not work as the bindings doc is technically an "article" not a vignette - vignettes are distributed with the package but articles are just published on the website.  If you look in `r/R/vignettes`, every `.Rmd` file in that directory is a vignette, but anything in a subdirectory (such as `developers/`) is an article.

Reply:
Ok, thanks for the info! Will only keep the link to the article then.